queue_services/entity-emailer/tests/unit/test_worker.py

```diff
@@ -466,3 +468,58 @@ def test_send_email_with_incomplete_payload(app, session, email_msg):
         worker.send_email(email_msg, None)
 
     assert 'Unsuccessful sending email' in str(excinfo)
+
+
+@pytest.mark.parametrize(['test_name', 'exception', 'furnishing_name', 'expected_furnishing_status'], [
+    ('Will be failed with invalid furnishing_name', QueueException, 'INVALID_NAME', 'QUEUED'),
```

Comment:
As we had talked about, we don't be throwing a `QueueException` when there is no matching furnishing name.

The code will just skip the message and log a message.